README.md

```diff
@@ -28,12 +28,12 @@ INSTALL
 Copy the statsd library from the .m2 folder to cassandra/lib.
 Add the following to your cassandra startup script:
 
-Copy the agent-1.1.jar to a new directory cassandra/plugins
+Copy the agent-1.2.jar to a new directory cassandra/plugins
 
 Change cassandra startup to add this agent. This can be done in
 a stock install by adding the following to /etc/default/cassandra:
```

Comment:
Agreed. The reason for putting it somewhere else is that chef manages the cassandra-env.sh file. More specifically, it's the community cookbook that manages that file, so adding that line in there was much more challenging. JMX settings are all controlled by the community cookbook.

Also, I phrased the README to say that it "can be done" this way but doesn't mean there aren't other ways to do it :)